src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,45 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class DeleteHistories {
+	private final List<DeleteHistory> deleteHistories;
+
+	public DeleteHistories() {
+		this.deleteHistories = new ArrayList<>();
+	}
+
+	public DeleteHistories(DeleteHistory deleteHistory) {
+		List<DeleteHistory> deleteHistories = new ArrayList<>();
+		deleteHistories.add(deleteHistory);
+		this.deleteHistories = deleteHistories;
+	}
+	//for test
+	DeleteHistories(List<DeleteHistory> deleteHistories) {
+		this.deleteHistories = deleteHistories;
+	}
+
+	public DeleteHistories add(DeleteHistory deleteHistory) {
+		this.deleteHistories.add(deleteHistory);
+		return this;
+	}
+
+	public List<DeleteHistory> transformToCollection() {
+		return deleteHistories;
```

Comment:
원본은 그대로 두고, 불변 객체를 새로 만들어 전달하는 방법도 추천드려요. 

```suggestion
            return Collections.unmodifiableList(deleteHistories);